wallet/src/Pos/Wallet/Web/Mode.hs

```diff
@@ -321,13 +322,6 @@ instance MonadKeysRead WalletWebMode where
 instance MonadKeys WalletWebMode where
     modifySecret = modifySecretDefault
 
-instance HasConfigurations => MonadWalletSendActions WalletWebMode where
-    sendTxToNetwork tx = do
-        saVar <- view wwmcSendActions_L
-        saMB <- atomically $ STM.tryReadTMVar saVar
-        let sa = fromMaybe (error "Wallet's SendActions isn't initialized") saMB
-        submitTxRaw (enqueueMsg sa) tx
```

Comment:
The existence of this class and this instance is a huge red flag. It's gone now.